File: Maximum Number of Coins You Can Get.js
var maxCoins = function(piles) {
 piles.sort((a, b) => b - a);
    
    let sum = 0,
	    startindex = 1,
		endIndex= 2 * piles.length / 3;
    while (startindex  < endIndex) {
        sum += piles[startindex];
        startindex += 2;
    }
    return sum ;
};

File: Maximum Number of Coins You Can Get.py
class Solution(object):
    def maxCoins(self, piles):
        piles.sort(reverse=True)
        sum = 0
        start_index = 1
        end_index = int(2 * len(piles) / 3)

        while start_index < end_index:
            sum += piles[start_index]
            start_index += 2

        return sum
